src/components/PasswordInput.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+    <label>
+        <div class="input__wrapper">
+            <span
+                v-if="label"
+                class="base-input__label">
+                {{ label }}
+            </span>
+            
+            <input 
+                :value="value"
+                @input="$emit('input', $event.target.value)"
+                :placeholder="placeholder"
+                :type="changeType"
+                class="base-input__input"> 
+                
+                <button title="Show password" class="button__password" @click="ShowHidePassword">
+                    <img src="@/assets/eyeCrossed.svg" alt="showPassword" :class="visible">
+                    <img src="@/assets/eye.svg" alt="hidePassword" class="hide" :class="unvisible">
```

Comment:
Instead of changing class it is much easier to use `v-if` or `v-show` directly based on `showPassword` state.

The difference is that `v-if` can actually adds or removes the element, but `v-show` toggles `display: none` (just as your class does 😉)

https://v2.vuejs.org/v2/guide/conditional.html#v-if
https://v2.vuejs.org/v2/guide/conditional.html#v-show

Reply:
Like this? I was reading about this later in the day and was thinking about using it. 

```
<img src="@/assets/eyeCrossed.svg" alt="showPassword" v-if="!showPassword" >
<img src="@/assets/eye.svg" alt="hidePassword" v-else>
```

And thank you!